presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/TestElasticsearchIntegrationSmokeTest.java

```diff
@@ -57,12 +57,15 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        elasticsearch = new ElasticsearchServer();
+        elasticsearch = new ElasticsearchServer("docker.elastic.co/elasticsearch/elasticsearch-oss:6.0.0", ImmutableMap.of());
```

Comment:
shall we make it a constant or configurable?
`docker.elastic.co/elasticsearch/elasticsearch-oss:6.0.0`

Reply:
kindly ping

Reply:
Made it a constant. Thanks for your reviews.